File: src/classes/api/req_res/update-notification-response.ts
'use strict'

import { Notification } from '@/classes/datas/notification'
import { GkillAPIResponse } from '../gkill-api-response'

export class UpdateNotificationResponse extends GkillAPIResponse {

    updated_notification: Notification

    constructor() {
        super()
        this.updated_notification = new Notification()
    }

}

File: src/server/gkill/api/req_res/update_notification_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type UpdateNotificationResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	UpdatedNotification *reps.Notification `json:"updated_notification"`
}
